guides/WRITING-DOCUMENTATION.md

```diff
@@ -2,25 +2,79 @@
 
 Documentation for the design system's components are automatically generated from their CSS and JS comments. In order for this to work, you need to follow some conventions set by the tools we use to generate the documentation: [KSS](https://github.com/kss-node/kss-node) and [react-docgen](https://github.com/reactjs/react-docgen).
 
-## File naming
+- [Directory structure](#directory-structure)
+  - [The docs folder structure](#the-docs-folder-structure)
+  - [The src folder structure](#the-src-folder-structure)
+- [Component documentation page formatting](#component-documentation-page-formatting)
+  - [Heading (required)](heading-required)
+  - [Description](#description)
+  - [@Flags](#flags)
+  - [Modifiers](#modifiers)
+  - [Markup](#markup)
+  - [Style guide](#style-guide)
+  - [Guidance](#guidance)
+- [Sample component documentation page](#sample-component-documentation-page)
+- [Documenting React components](#documenting-react-components)
+  - [Description](#description-1)
+  - [PropTypes](#proptypes)
+- [General content page](#general-content-page)
+  - [Front-matter](#front-matter)
+  - [Page content](#page-content)
+
+## Directory structure
 
-Name and structure components as follows in order for docs generation to work as expected.
+```
+child design system root
+├── `docs`    Directory where documentation and code examples are located
+└── `src`     Directory where component code, styles, fonts, and images are located
+```
+
+### The docs folder structure
+
+**Note:** Name and structure components as follows in order for the documentation site generation to work as expected.
+
+For a component named "Foo", you would have the following:
+
+```
+child design system root
+  └── docs
+      └── src
+          └── pages
+              └── components
+                  └── Foo                     Directory for component documentation content and code examples
+                      ├── Foo.docs.scss       Component documentation wirtten in KSS
+                      ├── Foo.example.jsx     React component example
+                      └── Foo.example.html    HTML example
+```
+
+### The src folder structure
```

Comment:
I dont think this is necessary to include this section, as it doesnt have to do with writing documentation